lib/utils/hasEmptyBlock.js

```diff
@@ -1,13 +1,13 @@
 'use strict';
 
+const hasBlock = require('./hasBlock');
+
 /**
  * Check if a statement has an empty block.
  *
  * @param {import('postcss').Rule | import('postcss').AtRule} statement - postcss rule or at-rule node
  * @return {boolean} True if the statement has a block and it is empty
  */
 module.exports = function hasEmptyBlock(statement) {
-	return (
-		statement.nodes !== undefined && statement.nodes.length === 0 // has block
-	); // and is empty
+	return hasBlock(statement) && statement.nodes.length === 0;
```

Comment:
[note] It should be more readable and less duplicated thanks to `hasBlock`:

https://github.com/stylelint/stylelint/blob/736bc297da67b1bb520ad4ec03ac5445fb1ae178/lib/utils/hasBlock.js#L9-L11